### PR TITLE
User specific view from Messaging extension: NodeJS

### PR DESCRIPTION
**Pull request by Prithvi-MSFT**

### CODE REVIEW

File: samples/bot-sequential-flow-adaptive-cards/nodejs/Readme.md [hunk 1]
```diff
@@ -138,6 +138,54 @@ List Incidents
 ![image](https://user-images.githubusercontent.com/85108465/123595684-d0d9ab80-d80e-11eb-9798-82f535ba6486.png)
 ![image](https://user-images.githubusercontent.com/85108465/123769386-e28e8200-d8e6-11eb-96e0-3d1f8365c7ef.png)
 
+## Workflow for ME Interaction
```
Comment (v-chetsh) [inline samples/bot-sequential-flow-adaptive-cards/nodejs/Readme.md:141]: Messaging Extension

File: samples/bot-sequential-flow-adaptive-cards/nodejs/server/bot/botActivityHandler.js [hunk 2]
```diff
@@ -27,12 +28,113 @@ class BotActivityHandler extends TeamsActivityHandler {
...
+                });
+
+                const incidentCard = CardFactory.adaptiveCard(await adaptiveCards.incidentListCard(choiceset));
+
+                return adaptiveCards.invokeIncidentTaskResponse("Select incident", incidentCard);
+            }
+
+            var incidentData = context.activity.value.data;
+            const incident = incs.find(inc => inc.id == incidentData.incidentId);
+            var card1 = CardFactory.adaptiveCard(await adaptiveCards.refreshBotCard(incident));
```
Comment (v-chetsh) [inline samples/bot-sequential-flow-adaptive-cards/nodejs/server/bot/botActivityHandler.js:52]: Put a meaningful name and don't put 1 or 2 like while creating variables.

File: samples/bot-sequential-flow-adaptive-cards/nodejs/server/bot/botActivityHandler.js [hunk 3]
```diff
@@ -27,12 +28,113 @@ class BotActivityHandler extends TeamsActivityHandler {
...
+                            },
+                            {
+                                type: 'TextBlock',
+                                text: 'Please create a incident using bot.',
+                                size: "medium",
+                                weight: "bolder"
+                            }
+                        ]
+                    });
+                    return adaptiveCards.invokeTaskResponse("No Incident found", noIncidentFound);
```
Comment (v-chetsh) [inline samples/bot-sequential-flow-adaptive-cards/nodejs/server/bot/botActivityHandler.js:86]: One line space before it.

File: samples/bot-sequential-flow-adaptive-cards/nodejs/server/models/adaptiveCard.js [hunk 4]
```diff
@@ -1447,6 +1498,168 @@ const refreshInc = async (user, action) => {
...
+                id: 'inc_title',
+                type: 'RichTextBlock',
+                inlines: [
+                    {
+                        type: 'TextRun',
+                        text: inc.title
+                    }
+                ]
+            },
+
```
Comment (v-chetsh) [inline samples/bot-sequential-flow-adaptive-cards/nodejs/server/models/adaptiveCard.js:1547]: Remove extra empty space and check if it's aligned properly.

File: samples/bot-sequential-flow-adaptive-cards/nodejs/server/models/adaptiveCard.js [hunk 5]
```diff
@@ -1447,6 +1498,168 @@ const refreshInc = async (user, action) => {
...
+                                        text: inc.subCategory,
+                                        horizontalAlignment: 'Center'
+                                    }
+                                ]
+                            }
+                        ]
+                    }
+                ]
+            },
+
```
Comment (v-chetsh) [inline samples/bot-sequential-flow-adaptive-cards/nodejs/server/models/adaptiveCard.js:1609]: Remove extra empty space and check if it's aligned properly.

**Review by v-chetsh — COMMENTED** [review]
Reviewed and provided comments.

**Review by v-chetsh — COMMENTED** [review]
Changes looks fine, approving it.

**Review by Wajeed-msft — APPROVED** [review]

**Review by AbhijeetThakur-msft — APPROVED** [review]

**Pull request merged**